packages/commonwealth/client/scripts/views/modals/new_snapshot_proposal_modal.tsx

```diff
@@ -23,18 +23,25 @@ export const NewSnapshotProposalModal = ({
   onModalClose,
 }: NewSnapshotProposalModalProps) => {
   const [selectedSnapshotId, setSelectedSnapshotId] = useState<string | null>(
-    null
+    null,
   );
+  const handleSelect = (item) => {
+    setSelectedSnapshotId(item.value);
+    const init = async () => {
+      await app.snapshot.init(item.value);
+    };
+    init();
+  };
 
   const snapshotOptions = useMemo(
     () =>
       app.chain?.meta.snapshot.map((snapshot) => ({
         value: snapshot,
         label: snapshot,
       })) || [],
-    []
+    [],
   );
-
+  console.log('snapshotOptions', snapshotOptions);
```

Comment:
Just pointing out this console log since I'm not sure if you meant to leave it here.

Reply:
@Miaplacidus I did not mean to leave that in, good catch thank you. Just pushed a change.